orange/src/test/java/com/hartwig/hmftools/orange/algo/purple/PurpleInterpreterTest.java

```diff
@@ -80,37 +85,212 @@ public void canImplyDeletionsFromBreakends()
                 .type(LinxBreakendType.DEL)
                 .undisruptedCopyNumber(0.4)
                 .build();
+        DriverGene driverGene = createDriverGene(DriverGeneGermlineReporting.WILDTYPE_LOST);
 
         StructuralVariant shortSv = create("vcf id 1", 10, 20);
         LinxSvAnnotation svAnnotation = LinxOrangeTestFactory.svAnnotationBuilder().svId(1).vcfId(shortSv.id()).build();
 
         List<GermlineDeletion> impliedMatch = PurpleInterpreter.implyDeletionsFromBreakends(Lists.newArrayList(),
                 Lists.newArrayList(left, right),
                 Lists.newArrayList(shortSv),
-                Lists.newArrayList(svAnnotation));
+                Lists.newArrayList(svAnnotation),
+                Lists.newArrayList(driverGene));
 
         assertEquals(1, impliedMatch.size());
         GermlineDeletion deletion = impliedMatch.get(0);
         assertEquals(TEST_GENE, deletion.GeneName);
         assertEquals(GermlineStatus.HOM_DELETION, deletion.TumorStatus);
+        assertEquals(0.4, deletion.TumorCopyNumber, EPSILON);
+        assertEquals(10, deletion.RegionStart);
+        assertEquals(20, deletion.RegionEnd);
 
         GermlineDeletion existingDeletion = GermlineDeletionTestFactory.create(TEST_GENE);
         List<GermlineDeletion> impliedExisting = PurpleInterpreter.implyDeletionsFromBreakends(Lists.newArrayList(existingDeletion),
                 Lists.newArrayList(left, right),
                 Lists.newArrayList(shortSv),
-                Lists.newArrayList(svAnnotation));
+                Lists.newArrayList(svAnnotation),
+                Lists.newArrayList(driverGene));
 
         assertEquals(0, impliedExisting.size());
 
         StructuralVariant longSv = create("vcf id 1", 10, 200000);
         List<GermlineDeletion> impliedTooLong = PurpleInterpreter.implyDeletionsFromBreakends(Lists.newArrayList(existingDeletion),
                 Lists.newArrayList(left, right),
                 Lists.newArrayList(longSv),
-                Lists.newArrayList(svAnnotation));
+                Lists.newArrayList(svAnnotation),
+                Lists.newArrayList(driverGene));
 
         assertEquals(0, impliedTooLong.size());
     }
 
+    @Test
+    public void canImplyDeletionsFromBreakendsWildtypeLostHeterozygous()
```

Comment:
What does this function test exactly? No deletions are implied by this config?

Reply:
Renamed for clarity.